core/workspace_svg.js

```diff
@@ -268,6 +268,13 @@ Blockly.WorkspaceSvg.prototype.flyoutButtonCallbacks_ = {};
  */
 Blockly.WorkspaceSvg.prototype.toolboxCategoryCallbacks_ = {};
 
+/**
+ * Developers may define this function to add custom menu options to the
+ * workspace's context menu or edit the workspace-created set of menu options.
+ * @param {!Array.<!Object>} options List of menu options to add to.
+ */
+Blockly.WorkspaceSvg.prototype.customContextMenu = null;
```

Comment:
CustomContextMenu is a noun phrase. As a function, I'd expect a verb phrase. configureWorkspaceMenu()?

Reply:
How about configureContextMenu, or configureContextMenuFn?

I was matching the API that we have for blocks, but I agree that it's not a great name.